scripts/import/import_EPMC.pl

```diff
@@ -122,15 +121,19 @@
 
 else{
     die "Type $type is not recognised - must be EPMC or UCSC\n";
-}
-
+} 
 
 ## create report on curent status
-report_summary($dba, $species);
+my $title_null = report_summary($dba, $species);
 
 ## add run date to meta table
 update_meta($dba, $type);
 
+## clean publications after import
+if(defined $clean && $clean == 1){ 
+  clean_publications($dba, $title_null);
+}
+
```

Comment:
We hold the information that a variant is cited as an evidence type on variation and variation feature. The line - 

update_evidence($dba) unless defined $no_evidence;

needs to move to after the citation deletion, so it is derived from the correct data.

Reply:
I moved this line to after clean_publications. Thanks!